devicetypes/smartthings/zigbee-dimmer-with-motion-sensor.src/zigbee-dimmer-with-motion-sensor.groovy

```diff
@@ -25,6 +25,7 @@ metadata {
 		capability "Health Check"
 
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0500, 0702, 0B05, FC01", outClusters: "0019", manufacturer: "sengled", model: "E13-N11", deviceJoinName: "Sengled Element Classic PAR38 Motion Sensor"
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, 0702, 0B05,0001,0402,0500,0020,0B0", outClusters: "0019", manufacturer: "sengled", model: "E13-A21", deviceJoinName: "Sengled Element Plus"
```

Comment:
There are some issues with formatting and length of the inClusters. I'd suggest using this simpler format though since the cluster list is getting so long:

`fingerprint manufacturer: "sengled", model: "E13-A21", deviceJoinName: "Sengled Element Plus"`. 

Also please double check the join name is correct. The name is much more generic than the previous fingerprint.

Reply:
Updated the incluster details and join name.
Please review and merge.